Lib/pdb.py

```diff
@@ -1352,6 +1352,9 @@ def do_longlist(self, arg):
         breaklist = self.get_file_breaks(filename)
         try:
             lines, lineno = inspect.getsourcelines(self.curframe)
+            # inspect.getsourcelines() returns lineno = 0 for
+            # module-level frame which breaks our code print line number
+            lineno = max(1, lineno)
```

Comment:
Maybe extract these 4 lines to a helper function so we don't have them repeated twice?

Reply:
So like a `getsourcelines()` function inside `pdb` like we got rid of a couple of weeks ago :).

It's actually just one line of "real" extra code and also hopefully temporary. I can do the abstraction, it just seems like a very small helper function :)